### PR TITLE
first version of hibernate order homework

### PR DESCRIPTION
**Pull request by oleksiistepaniak**

### CODE REVIEW

File: src/main/java/mate/academy/Main.java [hunk 1]
```diff
@@ -2,17 +2,35 @@
...
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
+import mate.academy.service.OrderService;
+import mate.academy.service.ShoppingCartService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static final MovieService movieService =
```
Comment (De1eF) [inline src/main/java/mate/academy/Main.java:20]: Either move initializations closer to their first use, or make them class variables

File: src/main/java/mate/academy/model/Order.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    private Long id;
+    @OneToMany
+    @JoinTable(name = "orders_tickets",
+            joinColumns = @JoinColumn(name = "order_id"),
+            inverseJoinColumns = @JoinColumn(name = "ticket_id"))
+    private List<Ticket> tickets;
+    @Column(name = "order_date")
+    private LocalDateTime orderDate;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private User user;
```
Comment (De1eF) [inline src/main/java/mate/academy/model/Order.java:31]: Should be named via @JoinColumn

**Review by De1eF — APPROVED** [review]
I made the same mistake)

File: src/main/java/mate/academy/service/impl/OrderServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+    @Inject
+    private ShoppingCartService shoppingCartService;
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        Order order = new Order();
+        order.setTickets(shoppingCart.getTickets());
```
Comment (petro-ushchuk) [inline src/main/java/mate/academy/service/impl/OrderServiceImpl.java:24]: Check common mistakes and fix this line

**Review by petro-ushchuk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**